app/fullService/api/getAccount.ts

```diff
@@ -41,12 +41,11 @@ export const isAccountSynced = async ({ accountId }: GetAccountParams): Promise<
     throw new Error('Failure to retrieve data.');
   } else {
     const { selectedAccount } = store.getState();
-    const isServerSynced =
-      Number(result.account.nextBlockIndex) === Number(result.networkBlockHeight);
-    const isClientSynced =
-      Number(selectedAccount.balanceStatus.accountBlockHeight) ===
-      Number(result.networkBlockHeight);
-    return isServerSynced && isClientSynced;
+    const isWalletSynced =
```

Comment:
I wonder if this shouldn't be compared to the networkBlockHeight in online mode and localBlockHeight in offline mode. Any thoughts?

Has this been tested with an offline wallet at all?

Reply:
oh, good catch! I hadn't thought of offline mode